docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,273 @@
+---
+sidebar_label: "Audit Scanner"
+title: "Audit Scanner"
+---
+
+# Audit Scanner
+
+Beginning with version `v1.7.0`, Kubewarden has added a new feature called the
+Audit Scanner. This component constantly checks if the resources running in the
+cluster adhere to the applied policies. It becomes especially handy when
+policies are deployed or updated, potentially modifying what was previously
+allowed in the cluster. The Audit Scanner provides operators with a tool to
+consistently verify whether any resources in the cluster violate the existing
+security and compliance rules. This can happen if the resources were deployed
+prior to the policies or if the policies were updated afterwards.
+
+To illustrate the usage of the audit scanner in Kubewarden, let's consider the
+following scenario:
+
+Assume you have deployed a set of policies to enforce specific rules and
+validations on your Kubernetes cluster. However, some resources were deployed
+before these policies were implemented, and you want to ensure that these
+resources comply with the newly deployed policies.
+
+To address this need, Kubewarden introduced the audit scanner component
```

Comment:
this paragraph is a bit of a repeat of the first paragraph.

Maybe I would try to simplify the scenario example, and summarize it on 2 lines or so.